browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -223,6 +223,13 @@ void BraveToolbarView::Init() {
   }
 #endif
 
+  // Locate avatar button right next to location bar in private/tor window.
+  auto location_bar_index = container_view->GetIndexOf(location_bar_);
+  if (location_bar_index && browser_->profile()->IsIncognitoProfile()) {
+    container_view->ReorderChildView(GetAvatarToolbarButton(),
```

Comment:
Q. I'm wondering if avatar button is guaranteed to exist regardless of browser window type, such as popup.